Cli-Shared/Program.cs

```diff
@@ -1113,6 +1215,44 @@ private static bool QueryCredentials(OperationArguments operationArguments)
                     }
                     break;
 
+                case AuthorityType.Bitbucket:
+					{
```

Comment:
nit: there's some kind of indentation fiasco happening here - can you please correct? Thanks.